docs/quickstarts/vue.mdx

```diff
@@ -180,3 +180,22 @@ Clerk's [Vue SDK](/docs/references/vue/overview) provides prebuilt components an
 
   Visit your app's homepage at [`http://localhost:5173`](http://localhost:5173). Sign up to create your first user.
 </Steps>
+
+## More resources
+
+Learn more about Clerk components, how to customize them, and how to use Clerk's client-side helpers using the following guides.
+
+<Cards>
+  - [Prebuilt components](/docs/components/overview)
+  - Learn more about Clerk's suite of components that let you quickly add authentication to your app.
+
+  ---
+
+  - [Customization & localization](/docs/customization/overview)
```

Comment:
I will do a follow up PR to this to add Vue and Nuxt sections of our customization and localization pages